README.md

```diff
@@ -1,57 +1,85 @@
 # merge-schedule-action
 
-> GitHub Action to merge pull requests on a scheduled day
+> GitHub Action to merge pull requests on a scheduled date and time.
 
-## Usage
+## How to use merge-schedule-action
+
+### Github Action Example
 
 Create `.github/workflows/merge-schedule.yml`
 
 ```yml
-name: Merge Schedule
+name: Scheduled PR Merge
 on:
   pull_request:
     types:
       - opened
       - edited
       - synchronize
+      - labeled
+      - unlabeled
   schedule:
     # https://crontab.guru/every-hour
     - cron: 0 * * * *
 
 jobs:
-  merge_schedule:
+  scheduled_PR_Merge_job:
+    name: Scheduled PR Merge Job
     runs-on: ubuntu-latest
+    outputs:
+      mergedPRs: ${{ steps.mainActionStep.outputs.MPR_LIST }}
+      mergeFailPR: ${{ steps.mainActionStep.outputs.MC_PR_NO }}
     steps:
-      - uses: gr2m/merge-schedule-action@v1
+      - id: mainActionStep
+        uses: VaibhavChidrawar/merge-schedule-action@env-var
         with:
           # Merge method to use. Possible values are merge, squash or
           # rebase. Default is merge.
-          merge_method: squash
+          merge_method: merge
```

Comment:
let's keep this as squash

Reply:
in general, we should recommend using squash instead of merge

Reply:
Done.